src/operator/ResourcesCloudProvider.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  ComponentColor,
+  DropdownMenuTheme,
+  MultiSelectDropdown,
+} from '@influxdata/clockface'
+import React, {FC, useContext} from 'react'
+import {OperatorContext} from './context/operator'
+import {OperatorRegions} from 'src/types'
+
+const ResourcesCloudProvider: FC = () => {
+  const {
+    providerInfo,
+    providers,
+    regions,
+    setProviders,
+    setRegions,
+  } = useContext(OperatorContext)
+
+  const hasSelectedProvider = providers.length > 0
+
+  const availableRegions: OperatorRegions = providers
+    .map(p => getRegions(p, providerInfo.regions))
+    .flat()
+
+  const providerOptions = providerInfo.providers.map(p => p.provider)
+  const regionOptions = availableRegions.flat().map(r => r.region)
+
+  const handleSelectProvider = (selectedOption: string): void => {
+    setRegions([])
+    providers.includes(selectedOption)
+      ? setProviders(prev => prev.filter(x => x !== selectedOption))
+      : setProviders(prev => [selectedOption, ...prev])
+  }
+
+  const handleSelectRegion = (selectedOption: string): void =>
+    regions.includes(selectedOption)
+      ? setRegions(prev => prev.filter(x => x !== selectedOption))
+      : setRegions(prev => [selectedOption, ...prev])
+
+  return (
+    <>
+      <MultiSelectDropdown
+        style={{width: '220px', marginRight: '10px'}}
```

Comment:
Is this component going to be rendered on every operator page? If so, I think this style object literal should be set into a variable, since object literals passed as props to components cause unnecessary renders. If not, it's probably not a huge deal, but it's a nice-to-have.

https://www.debugbear.com/blog/react-rerenders#passing-objects-as-props